### PR TITLE
[#121573867] add view & tests for "framework agreement signature" page

### PR DESCRIPTION
**Pull request by risicle**
This is the view & view tests for this feature, empty template due to not having been written yet. Will need participation from someone else there.

### CODE REVIEW

**Comment by risicle** [comment]
(bah hold on think i pushed the wrong version)

**Comment by TheDoubleK** [comment]
Is this still on hold, or should someone take a look at it now?

**Comment by risicle** [comment]
It's on hold in that there's no html for it yet.

**Comment by risicle** [comment]
Ok, I've rebased this - should be ready for review...

File: app/main/views/frameworks.py [hunk 1]
```diff
@@ -122,23 +122,30 @@ def framework_dashboard(framework_slug):
...
-        abort(404)
 
-    drafts, complete_drafts = get_drafts(data_api_client, framework_slug)
-    applied_lot_slugs = frozenset(draft["lotSlug"] for draft in complete_drafts)
+        abort(404)
+    print('FRAMEWORK HAS FRAMEWORK INFO')
+    # drafts, complete_drafts = get_drafts(data_api_client, framework_slug)
+    # applied_lot_slugs = frozenset(draft["lotSlug"] for draft in complete_drafts)
 
+    print('RENDERING TEMPLATE')
```
Comment (risicle) [inline app/main/views/frameworks.py]: Hold on @laurenceberry - did you mean to include all these changes to the view? What's the intent here? (apart from the debugging lines)

Comment (allait) [inline app/main/views/frameworks.py]: Yeah, the comments have to go

**Comment by allait** [comment]
Do we need something else to replace the placeholders in the template with the actual data?

**Comment by pcraig3** [comment]
🔪 🔪 🔪

**Comment by TheDoubleK** [comment]
I think this PR can be closed now we are doing PDFs (but let's keep the branch intact, just in case...)

**Comment by allait** [comment]
Yeah, I think this can be closed. We'll move the page template to the agreements repo to use for PDF generation.

**Pull request closed without merging**